### PR TITLE
ResultPage tweaks!

### PR DESCRIPTION
**Pull request by Nirzhuk**
Tweak design.

### CODE REVIEW

File: src/results/PersonalBenefits.js [hunk 1]
```diff
@@ -88,10 +88,10 @@ class PersonalBenefits extends React.Component<Props> {
                   </span>{" "}
                     € / {benefit.periode}
                   </Grid>
-                  <Grid item xs={6} sm={3}>
+                  <Grid className={"Separator"}item xs={6} sm={3}>
```
Comment (jvalduvieco) [inline src/results/PersonalBenefits.js]: no need to use `{}`

File: src/results/PersonalBenefits.js [hunk 2]
```diff
@@ -88,10 +88,10 @@ class PersonalBenefits extends React.Component<Props> {
...
                     <Link to={benefit.url}>
                       <button
-                          style={{float: "right"}}
+                          style={{float: "right", marginTop: 17}}
```
Comment (jvalduvieco) [inline src/results/PersonalBenefits.js]: Are you sure you want to embed style in the element?

File: src/shared/OpenFiscaAPIClient.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ class OpenFiscaAPIClient {
       isDevel = isDevelopment;
     }
     if (isDevel) {
-      this.url = this.developmentURL;
+      this.url = this.productionURL;
```
Comment (jvalduvieco) [inline src/shared/OpenFiscaAPIClient.js]: Do not commit this plz.

File: src/results/PersonalBenefits.js (hunk 1 above)
Comment (jvalduvieco) [reply src/results/PersonalBenefits.js]: As you are not reusing this class anywhere els I'd prefer to have classnames that identify the element like `ResultsPageMoreInfoButtonContainer`

File: src/results/PersonalBenefits.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ import {Map} from "immutable";
 import type {AdultId} from "../adults/AdultsTypes";
 import {Adult} from "../adults/AdultsTypes";
 import {Grid} from "material-ui";
-
+import { } from "material-ui-icons"
```
Comment (jvalduvieco) [inline src/results/PersonalBenefits.js]: huh?

File: src/results/PersonalBenefits.js [hunk 5]
```diff
@@ -71,13 +71,13 @@ class PersonalBenefits extends React.Component<Props> {
           <Grid
               className="ResultPage"
               container
-              justify={"space-between"}
+              justify={"center"}
```
Comment (jvalduvieco) [inline src/results/PersonalBenefits.js]: no need `{}`

File: src/results/PersonalBenefits.js (hunk 1 above)
Comment (jvalduvieco) [reply src/results/PersonalBenefits.js]: space between `className` and `item`. Also `item` should be the first property.

File: src/results/PersonalBenefits.js (hunk 2 above)
Comment (Nirzhuk) [reply src/results/PersonalBenefits.js]: It got already a class so I don't know really.

**Pull request merged**